MapboxCoreNavigation/RouteController.swift

```diff
@@ -316,6 +333,35 @@ open class RouteController: NSObject {
     }
     
     /**
+     Checks whether a new location is quality by comparing it to the previous location.
+     
+     We're attempting to throw out coordinates that have a large difference in location or time.
+     */
+    func isQualityLocation(location: CLLocation?) -> Bool {
+        
+        // Any time we return in this func, reset the values.
+        defer {
+            if let location = location {
+                if let lastLocation = lastQualityLocationInterval {
+                    lastQualityLocationInterval?.speedDelta = location.speed - lastLocation.location.speed
```

Comment:
Unless we take the absolute value of this difference somewhere, we’re going to toss out any location update that doesn’t represent a speedup of 3× or more. 💨